### PR TITLE
feat(args): allow returning the bumped version

### PR DESCRIPTION
**Pull request by nappa85**
## Description

This PR adds a CLI command to simply return the bumped version, solves #361 

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (no code change)
- [ ] Refactor (refactoring production code)
- [ ] Other 

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] I have formatted the code with [rustfmt](https://github.com/rust-lang/rustfmt).
- [x] I checked the lints with [clippy](https://github.com/rust-lang/rust-clippy).
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: git-cliff/src/args.rs [hunk 1]
```diff
@@ -146,10 +146,13 @@ pub struct Opt {
...
 	)]
-	pub tag:          Option<String>,
+	pub tag:            Option<String>,
 	/// Bumps the version for unreleased changes.
 	#[arg(long, help_heading = Some("FLAGS"))]
-	pub bump:         bool,
+	pub bump:           bool,
+	/// Returns bumped version for unreleased changes.
+	#[arg(long, help_heading = Some("FLAGS"))]
+	pub bumped_version: bool,
```
Comment (orhun) [inline git-cliff/src/args.rs:155]: Not sure if I like the argument name. We can go with `--print-version` or something similar. What do you think?

File: git-cliff/src/lib.rs [hunk 2]
```diff
@@ -59,6 +62,11 @@ fn check_new_version() {
 	}
 }
 
+enum ProcessOutput<'a> {
+	Releases(Vec<Release<'a>>),
+	Version(String),
+}
```
Comment (orhun) [inline git-cliff/src/lib.rs:68]: Can you document the enum/fields for clarity?

File: website/docs/usage/examples.md [hunk 3]
```diff
@@ -32,6 +32,14 @@ Calculate and set the next semantic version (i.e. _bump the version_) for the un
 git cliff --bump
 ```
 
+Simply calculate and return next semantic version:
+
+```bash
+# Semver: {MAJOR}.{MINOR}.{PATCH}
+# "fix:" increments PATCH, "feat:" increments MINOR and "scope!" (breaking changes) increments MAJOR
+git cliff --bumped-version
+```
```
Comment (orhun) [inline website/docs/usage/examples.md:41]: We can merge this with this example above to avoid the duplicate explanation of SemVer.

**Review by orhun — CHANGES_REQUESTED** [review]
Had some comments - please check 🐻 

Can you also add [fixture tests](https://github.com/orhun/git-cliff/tree/main/.github/fixtures) for testing the output?

**Review by orhun — APPROVED** [review]
Thanks for your contribution! 💖

**Comment by nappa85** [comment]
> Thanks for your contribution! 💖

Thank you for your patience

**Pull request merged**